test/integration/apiserver/apiserver_test.go

```diff
@@ -3352,3 +3360,132 @@ func assertManagedFields(t *testing.T, obj *unstructured.Unstructured) {
 func int32Ptr(i int32) *int32 {
 	return &i
 }
+
+// TestDefaultStorageEncoding verifies the storage encoding of each resource.
```

Comment:
expand on details.  All built-ins should use protobuf unless we've had an accident.  Looks like have two existing accidents.